lit/docs/install/concourse-cli.lit

```diff
@@ -9,11 +9,20 @@ contains the following files:
 
 \codeblock{sh}{{
 concourse/bin/concourse
-concourse/bin/gdn            # on Linux
+concourse/bin/gdn            # Linux only
 concourse/fly-assets/...
-concourse/resource-types/... # on Linux
+concourse/resource-types/... # Linux only
 }}
 
+The Linux release is the largest among all the platforms because it contains
+some of the most commonly used resource types like the
+\link{git}{https://github.com/concourse/git-resource},
+\link{time}{https://github.com/concourse/time-resource/}, and
+\link{registry-image}{https://github.com/concourse/registry-image-resource/}
+resources. Resources only run on Linux workers, that's why the other platforms
```

Comment:
We could also point to the "Bundled resource types" section of any release newer than v6.7.3